src/libs/CurrencyUtils.ts

```diff
@@ -96,11 +96,11 @@ function convertToFrontendAmountAsInteger(amountAsInt: number): number {
  *
  * @note we do not support any currencies with more than two decimal places.
  */
-function convertToFrontendAmountAsString(amountAsInt: number | null | undefined): string {
+function convertToFrontendAmountAsString(amountAsInt: number | null | undefined, withDecimals = true): string {
     if (amountAsInt === null || amountAsInt === undefined) {
         return '';
     }
-    return convertToFrontendAmountAsInteger(amountAsInt).toFixed(2);
+    return convertToFrontendAmountAsInteger(amountAsInt).toFixed(withDecimals ? 2 : 0);
```

Comment:
We are implementing a [PR](https://github.com/Expensify/App/pull/43948/files) to make this change. In that PR, the decimal relies on the currency and I think it will make more sense

Reply:
so we will be able to get an amount without decimals then?

Reply:
I am not sure what you mean.

Reply:
In this [PR](https://github.com/Expensify/App/pull/43948/files), we will display the decimal relies on the current currency. Then do we need to apply your change here any more?

Reply:
yes, because we don't want to display any decimals here - it's for the limit step. I guess that after the changes from this PR you mentioned we will have 2 decimals for USD and for other currencies using 2 decimals.